packages/libs/react-ui/src/components/TextLink/TextLink.tsx

```diff
@@ -34,11 +35,18 @@ export type ITextLinkProps = Omit<AriaFocusRingProps, 'isTextInput'> &
  * @param className - additional class name
  * @param style - additional style
  * @param title - title to be shown as HTML tooltip
+ * @param withIcon - optionally show an link icon
  */
 
 const TextLink = forwardRef(
   (
-    { isCompact = false, children, className, ...props }: ITextLinkProps,
+    {
+      isCompact = false,
+      children,
+      className,
+      withIcon,
```

Comment:
Should this, for backwards compatibility reasons, default to true? 🤷‍♂️

Reply:
Technically yes, but as the component is still very new we were bold and changed it to how UX initially wanted it to work.

Reply:
So bold 💪 😬